index.html

```diff
@@ -371,6 +371,23 @@ <h2 class="logo-scratchx">ScratchX</h2>
                         </section>
                     </li>
                     <li>
+                        <a href="https://ericrosenbaum.github.io/spotify-extension/" target="_blank">
```

Comment:
👍  Sorry ... I read your note after reviewing the code. Sounds good to me. I think we may want to revisit this in the future but I'm ok to deviate for now.